templates/genie.yaml

```diff
@@ -167,9 +167,53 @@ Resources:
         Command:
           - python
           - database_to_staging.py
-          - Jan-2019
+          - Jul-2019
           - /root/cbioportal
-          - 6.2-consortium
+          - 7.1-consortium
+          - --pemFile
+          - /root/.ssh/genie.pem
+        Volumes:
+          - Host:
+              SourcePath: "/home/ec2-user/.ssh/genie.pem"
+            Name: geniePem
+          - Host:
+              SourcePath: "/home/ec2-user/.synapseCache"
+            Name: synapsecache
+        Environment:
+          - Name: GENIE_PASS
+            Value: (b'\x1b\x1f\xc8\xdb\x9f\x03\x0c\xf6\xd4\x95NI\x06\x97e\xda\x88i~\x89\x9a"\x19ON\x8c\x01\x9a\xb4\xaat\x970\xb0\xeel\xae\t\'\xdb\xf3XUQw\xdb\xe7\x11\xd3\x0bf\xc6\xaf\xae\x11\xe2\xa9W\x8b\x08\xcb\x92\xc5\x8f3-\xf8\x92[\x8b\xb8\xe0\xcc\x87\xc09\xa5\rw\x03\xbc\xa4\xa6Y\xa1\x87\x8d\xe6G\xba5\xbb\xa4\x84xl\xba\xa1\x05\xa6\xc7\x1d&N>\xb7\xf0\x1b\x19#E\xb1\xd5\xaf\xc3\xafU,\xfbb\x95\x9df\xeb\xfe&\x0f\xaa',)
+          - Name: GENIE_USER
+            Value: genieRunner
+        MountPoints:
+          - ContainerPath: "/root/.ssh/genie.pem"
+            ReadOnly: false
+            SourceVolume: geniePem
+          - ContainerPath: "/root/.synapseCache"
+            ReadOnly: false
+            SourceVolume: synapsecache
+      RetryStrategy:
+        Attempts: 3
+      Timeout:
+        AttemptDurationSeconds: 172800
```

Comment:
3 retries at 48 hrs each, that's 6 days.  just for one job? is that normal?

Reply:
The release jobs take anywhere from 24 to 48 hours.  The mutation processing is taking up to 4 or 5 days now.  The retries isn't based on timeout, but just on execution time.